project/plugins.sbt

```diff
@@ -6,10 +6,12 @@ addSbtPlugin("com.eed3si9n" % "sbt-buildinfo" % "0.7.0")
 
 addSbtPlugin("com.eed3si9n" % "sbt-assembly" % "0.14.5")
 
-libraryDependencies += "org.vafer" % "jdeb" % "1.3" artifacts (Artifact("jdeb", "jar", "jar"))
+libraryDependencies += "org.vafer" % "jdeb" % "1.5" artifacts (Artifact("jdeb", "jar", "jar"))
 
 addSbtPlugin("org.scalastyle" %% "scalastyle-sbt-plugin" % "1.0.0" excludeAll ExclusionRule(organization = "com.danieltrinh"))
 
 addSbtPlugin("org.scalariform" % "sbt-scalariform" % "1.8.1")
 
-addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.3")
\ No newline at end of file
+addSbtPlugin("com.gu" % "sbt-riffraff-artifact" % "1.1.3")
+
+addSbtPlugin("com.timushev.sbt" % "sbt-updates" % "0.3.3")
```

Comment:
Just curious, is this so we can do `> reload plugins` in sbt and have everything automagically update for us?

Reply:
If you run `sbt dependencyUpdates` it gives you a list of libraries that have newer versions (e.g. `com.typesafe.akka:akka-agent                   : 2.4.12   -> 2.4.20   -> 2.5.6`

I don't know if it can automagically update them - I suspect not

Reply:
That sounds better. The git repo for the library suggests it is capable of updating in-project (not global) plug-ins with that. Which concerned me a bit 😨 

I get a bunch of warnings, but then I get those on dotcom frontend too; 

```[warn] Found version conflict(s) in library dependencies; some are suspected to be binary incompatible:
[warn] 	* com.google.guava:guava:23.0 is selected over {22.0, 11.0.1, 13.0.1, 19.0}
[warn] 	    +- support-frontend:support-frontend_2.11:1.0-SNAPSHOT (depends on 23.0)
[warn] 	    +- com.gu:play-googleauth_2.11:0.7.0                  (depends on 19.0)
[warn] 	    +- com.gu:ophan-event-model_2.11:0.0.1                (depends on 19.0)
[warn] 	    +- com.google.gdata:core:1.47.1                       (depends on 13.0.1)
[warn] 	    +- com.google.http-client:google-http-client:1.11.0-beta (depends on 11.0.1)
[warn] 	    +- com.google.oauth-client:google-oauth-client:1.11.0-beta (depends on 11.0.1)
[warn] 	    +- com.typesafe.play:play_2.11:2.6.6                  (depends on 22.0)
[warn] 	* com.gu:fezziwig_2.11:0.7 is selected over 0.6
[warn] 	    +- support-frontend:support-frontend_2.11:1.0-SNAPSHOT (depends on 0.7)
[warn] 	    +- com.gu:acquisition-event-producer_2.11:2.0.1       (depends on 0.6)
[warn] 	* com.typesafe.scala-logging:scala-logging_2.11:3.7.2 is selected over 3.5.0
[warn] 	    +- support-frontend:support-frontend_2.11:1.0-SNAPSHOT (depends on 3.7.2)
[warn] 	    +- com.gu:acquisition-event-producer_2.11:2.0.1       (depends on 3.7.2)
[warn] 	    +- com.gu:identity-test-users_2.11:0.6                (depends on 3.5.0)
[warn] 	* xpp3:xpp3 is evicted completely
[warn] 	    +- com.google.http-client:google-http-client:1.11.0-beta (depends on 1.1.4c)
[warn] 	* com.typesafe.akka:akka-actor_2.11:2.5.6 is selected over {2.5.4, 2.4.19}
[warn] 	    +- com.typesafe.akka:akka-agent_2.11:2.5.6 ()         (depends on 2.5.6)
[warn] 	    +- com.typesafe.akka:akka-slf4j_2.11:2.5.4            (depends on 2.5.4)
[warn] 	    +- com.typesafe.play:play_2.11:2.6.6                  (depends on 2.5.4)
[warn] 	    +- com.typesafe.akka:akka-stream_2.11:2.5.4           (depends on 2.5.4)
[warn] 	    +- com.typesafe.akka:akka-parsing_2.11:10.0.10        (depends on 2.5.4)
[warn] 	* com.twitter:scrooge-core_2.11:4.20.0 is selected over 4.18.0
[warn] 	    +- com.gu:ophan-event-model_2.11:0.0.1                (depends on 4.20.0)
[warn] 	    +- com.gu:fezziwig_2.11:0.7                           (depends on 4.18.0)
[warn] 	* com.typesafe.play:play-json_2.11:2.6.6 is selected over {2.6.0, 2.4.3}
[warn] 	    +- com.typesafe.play:play_2.11:2.6.6                  (depends on 2.6.6)
[warn] 	    +- com.gu:ophan-event-model_2.11:0.0.1                (depends on 2.4.3)
[warn] 	    +- com.typesafe.play:play-ws-standalone-json_2.11:1.1.2 (depends on 2.6.0)
[warn] 	* org.scala-lang.modules:scala-java8-compat_2.11:0.8.0 is selected over 0.7.0
[warn] 	    +- com.typesafe.play:play-ahc-ws-standalone_2.11:1.1.2 (depends on 0.8.0)
[warn] 	    +- com.typesafe.akka:akka-actor_2.11:2.5.4            (depends on 0.7.0)
[warn] 	    +- com.typesafe.akka:akka-actor_2.11:2.5.6 ()         (depends on 0.7.0)
[warn] 	    +- com.typesafe.play:play_2.11:2.6.6                  (depends on 0.7.0)
[warn] 	    +- com.typesafe.play:play-streams_2.11:2.6.6          (depends on 0.7.0)
[warn] 	* com.typesafe.akka:akka-stream_2.11:2.5.4 is selected over {2.5.3, 2.4.19}
[warn] 	    +- com.typesafe.play:play-streams_2.11:2.6.6          (depends on 2.5.4)
[warn] 	    +- com.typesafe.akka:akka-http-core_2.11:10.0.10      (depends on 2.4.19)
[warn] 	    +- com.typesafe.play:play-ws-standalone_2.11:1.1.2    (depends on 2.5.3)
[warn] Run 'evicted' to see detailed eviction warnings```